src/directLine.ts

```diff
@@ -214,4 +226,92 @@ export class DirectLine implements IBotConnection {
             .delay(5 * 1000)
         );
     }
+
+    /**
+     * Gets an observable of streamURL by requestings a new one from the conversation APIs. 
+     * @return An Observable of stream url.
+     */
```

Comment:
We don't do this kind of documentation anywhere else in this project, so let's not do it here